### ISSUE TITLE
This email address does not belong to a Google Account

### ISSUE THREAD

**Issue by LIttleAncientForestKami** [mxrch/GHunt#144]
Hi there @mxrch and team and thank you for your work on this tool.

TL;DR: the tool reports that my cookies are fine and then it reports that given email doesn't belong to a google account (as per topic). I may be doing something wrong, but after encountering the tool I have done the following:

1. docker pull
2. ran hunt - learned about the cookies
3. got cookies from the account page (so I have the account, at least on accounts.google.com)
4. ran the check_and_gen script, which said cookies I gave it are valid
5. reran the hunt tool - got the titular message

There were no errors, no warnings:

```

     .d8888b.  888    888                   888    
    d88P  Y88b 888    888                   888    
    888    888 888    888                   888    
    888        8888888888 888  888 88888b.  888888 
    888  88888 888    888 888  888 888 "88b 888    
    888    888 888    888 888  888 888  888 888    
    Y88b  d88P 888    888 Y88b 888 888  888 Y88b.  
     "Y8888P88 888    888  "Y88888 888  888  "Y888
    
[-] This email address does not belong to a Google Account.
```

I used FF to get the cookies, am using freshest Docker image with the tool, am running Ubuntu OS. Any pointers?

**Comment by mxrch** [comment]
@LIttleAncientForestKami  Are you sure your email address exists ? Try contactsqueezie@gmail.com for example

**Comment by mxrch** [comment]
FYI : an email that looks obvious does not necesseraly exists. Like, google@gmail.com or trump@gmail.com.

**Comment by dirt3009** [comment]
Same for me on the latest version, using without Docker on Windows 10

Tried on contactsqueezie@gmail.com

**Comment by mxrch** [comment]
@dirt3009 
![image](https://user-images.githubusercontent.com/17338428/99076492-8414cd80-25bb-11eb-8a47-c96a470daae6.png)

It works for me so I don't know why you have this result.

**Comment by LIttleAncientForestKami** [comment]
Hi there @mxrch and thank you for your answers! 

To eliminate the problems with environment I used Docker and went with your suggestion:

```
$ _ docker run -v ghunt-resources:/usr/src/app/resources -ti mxrch/ghunt hunt.py contactsqueezie@gmail.com                                                                                                            [INSERT]

     .d8888b.  888    888                   888    
    d88P  Y88b 888    888                   888    
    888    888 888    888                   888    
    888        8888888888 888  888 88888b.  888888 
    888  88888 888    888 888  888 888 "88b 888    
    888    888 888    888 888  888 888  888 888    
    Y88b  d88P 888    888 Y88b 888 888  888 Y88b.  
     "Y8888P88 888    888  "Y88888 888  888  "Y888
    
[-] This email address does not belong to a Google Account.
```

I am certain that the address I used in the opening post exists, but it rendered same result. Anything I should do to debug this?

**Comment by mxrch** [comment]
@LIttleAncientForestKami try to regenerate your cookies, since this is the only difference between our installations.

**Comment by meytrix183** [comment]
Hi
I am having the same issue.
Tried with cookies of two accounts and tried with the docker version and the manual version.

**Comment by Zer0C00lness** [comment]
And in the end when your trying or Running the program is that when your problem occurs? Or is it when your commanding a task and your task is not being performed then?

€€/\/\@¡\¡’-|?0

**Comment by meytrix183** [comment]
> And in the end when your trying or Running the program is that when your problem occurs? Or is it when your commanding a task and your task is not being performed then? €€/\/\@¡\¡’-|?0
> […](#)
> On Dec 3, 2020, at 3:46 PM, meytrix183 ***@***.***> wrote: ﻿ Hi I am having the same issue. Tried with cookies of two accounts and tried with the docker version and the manual version. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe.

I have the same problem reported by @LIttleAncientForestKami , "This email address does not belong to a Google Account" and i know the account exists because its an account of mine.

**Comment by jcsimmons1** [comment]
Hello, great tool but I am getting the same issue. I have tried clearing out the cookies and regenerating them, etc. Worked fine until recently. Oddly enough, another online tool I use https://tools.epieos.com/google-account.php is having the same issue. I am trying my own gmail address in GHunt and this online too and I get "This email address does not belong to a Google Account". Thanks!

**Comment by rodrigobercini** [comment]
I'm having the same issue on Linux via Docker install. I'm 100% sure that the account exists.

**Comment by b19e** [comment]
I also have the same issue on ubuntu 20.04, via docker! Seem's to be random, I also tried using cookies from two different accounts. And a few different Gmail accounts, that belong to me.  " This email address does not belong to a Google Account "

**Comment by jcsimmons1** [comment]
Hey all - I had to reinstall it - required cookies were changed. It works fine now.

**Comment by mrme44** [comment]
Using the manual installation, I had to open up a new terminal after running the check_and_gen.py script.
